### PR TITLE
rename credentials fields

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

### CODE REVIEW

**Comment by leena-jawale** [comment]
There are these 3 fields in vmware need to change

```
"vcenter":              "vCenter",
"vmClusterName":        "cluster",
"datastore":            "defaultDatastore",
```

And also I am not seeing `osServicePrincipal.json` in PR

**Review by jeswanke — APPROVED** [review]

File: frontend/src/resources/provider-connection.ts [hunk 1]
```diff
@@ -17,25 +17,22 @@ export const ProviderConnectionDefinition: IResourceDefinition = {
...
-    awsSecretAccessKeyID?: string
+    aws_access_key_id?: string
+    aws_secret_access_key?: string
 
     baseDomainResourceGroupName?: string
-    clientId?: string
-    clientSecret?: string
-    subscriptionId?: string
-    tenantId?: string
+    ['osServicePrincipal.json']?: string
```
Comment (chrisahl) [inline frontend/src/resources/provider-connection.ts:24]: What is this [' '] notation for?


Other general comment is I would prefer to keep prefixes on variables to make it easier to know which provider they belong to.  Plus it seems to easy to have name collision with this change.

File: frontend/public/locales/en/create.json [hunk 2]
```diff
@@ -226,9 +226,9 @@
     "creation.ocp.network.interface": "Provisioning network interface",
     "creation.ocp.network.bridge": "Provisioning network bridge",
     "creation.ocp.external.bridge": "External network bridge",
-    "creation.ocp.vcenter": "vCenter",
+    "creation.ocp.vCenter": "vCenter",
     "creation.ocp.datacenter": "Datacenter",
-    "creation.ocp.datastore": "Default datastore",
+    "creation.ocp.defaultDatastore": "Default defaultDatastore",
```
Comment (chrisahl) [inline frontend/public/locales/en/create.json:231]: Shouldn't the text stay `Default datastore`?

File: frontend/src/resources/provider-connection.ts (hunk 1 above)
Comment (jamestalton) [reply frontend/src/resources/provider-connection.ts:24]: Any keys that have a '.' in them need to be quoted.
Renaming the keys to match the names used by hive.

**Review by chrisahl — APPROVED** [review]

**Pull request merged**